localstripe/resources.py

```diff
@@ -2075,12 +2074,19 @@ def __init__(self, customer=None, metadata=None, items=None,
         if items is None:
             items = [{ 'plan': plan, 'quantity': quantity }]
 
+        if type(trial_end) is str and trial_end == 'now':
```

Comment:
You don't need the `type(trial_end) is str and` here: `if trial_end == 'now'` is enough.